docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -186,6 +186,7 @@ In general, try to avoid reflection if possible. When using reflection, limit it
 - In some cases, you will be able to mechanically propagate warnings through your code without issues. Sometimes this will result in much of your public API being annotated with `RequiresUnreferencedCode`, which is the right thing to do if the library indeed behaves in ways that can't be understood statically by the trim analysis.
 - In other cases, you might discover that your code uses patterns that can't be expressed in terms of the `DynamicallyAccessedMembers` attributes, even if it only uses reflection to operate on statically known types. In these cases, you may need to reorganize some of your code to make it follow an analyzable pattern.
 - Sometimes the existing design of an API will render it mostly trim-incompatible, and you may need to find other ways to accomplish what it is doing. A common example is reflection-based serializers. In these cases, consider adopting other technology like source generators to produce code that is more easily statically analyzed.
+- The trimmer, by default, collapses warnings that happen in package references into a single IL2104. You can run with `/p:TrimmerSingleWarn=false` and get the full list of warnings which can help narrow down underlying issues.
```

Comment:
This doc mentions this option in "Trim warnings in apps" paragraph - look for "(You can also show detailed warnings for the library to see which parts of it are problematic.)".

Instead of adding it here, would it be better to make the earlier mention more verbose so that it is easier to find? I have completely missed it when I read the doc first time.